charts/Makefile

```diff
@@ -0,0 +1,54 @@
+REPO_BASE_DIR := $(shell git rev-parse --show-toplevel)
+
+include ${REPO_BASE_DIR}/scripts/common.Makefile
+include $(REPO_CONFIG_LOCATION)
+
+CONFIG_DIR := $(shell dirname $(REPO_CONFIG_LOCATION))
+CHART_DIRS := $(wildcard $(REPO_BASE_DIR)/charts/*/)
+
+.PHONY: .check-helmfile-installed
+.check-helmfile-installed:
+	@if ! command -v helmfile >/dev/null 2>&1; then \
+			echo "'helmfile' is not installed. Install it to continue ...";\
```

Comment:
you might want to add in the README how to install all these requirements?

Reply:
done